addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -170,6 +201,13 @@ export class RtcSession extends Record {
         this.localVolume = value;
     }
 
+    delete() {
+        const def = RtcSession.awaitedRecords.get(this.id);
+        def?.reject?.();
+        RtcSession.awaitedRecords.delete(this.id);
+        return super.delete(...arguments);
+    }
```

Comment:
How is it possible to await a record that gets deleted? Isn't the promise immediately resolved if the record already exists (or when it gets inserted)?

Reply:
@seb-odoo I was thinking that we could get a delete message on the bus (without being aware that this session even existed) and then reject early, but it's indeed not very useful, it will timeout anyways.

Reply:
`delete` from bus won't do anything if `get` doesn't return the record